Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.12-slim
+LABEL maintainer="testtest@gmail.com"
+ENV PYTHONUNBUFFERED=1
+COPY requirements.txt requirements.txt
```

Comment:
Consider using a multi-stage build to reduce the final image size. This can be done by installing dependencies in a separate stage and copying only the necessary files to the final stage.